core/src/build-staging/build-staging.ts

```diff
@@ -189,7 +189,7 @@ export class BuildStaging {
 
     // Source root must exist and be a directory
     let sourceStat = await statsHelper.extendedStat({ path: sourceRoot })
-    if (!sourceStat || !(sourceStat.isDirectory() || sourceStat.target?.isDirectory())) {
+    if (!sourceStat || !sourceStat.isDirectory()) {
```

Comment:
I think this one must have been an accident
```suggestion
    if (!sourceStat || !(sourceStat.isDirectory() || sourceStat.target?.isDirectory())) {
```